### PR TITLE
mithril - Change order in Hyperscript interface

### PR DESCRIPTION
**Pull request by foxdonut**
Change method order in Hyperscript interface so that IDE auto-suggest works on attributes.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## mithril ([<kbd>unpkg</kbd>](https://unpkg.com/browse/mithril@latest/))
was missing the following properties:
1. m
1. Fragment
1. vnode
1. PromisePolyfill
1. censor
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against f8d34623d0df33710ace140d505206336af602cd
</p>

File: types/mithril/index.d.ts [hunk 1]
```diff
@@ -42,18 +42,18 @@ declare namespace Mithril {
...
         /** Creates a virtual element (Vnode). */
         <Attrs, State>(
             component: ComponentTypes<Attrs, State>,
             attributes: Attrs & CommonAttributes<Attrs, State>,
             ...args: Children[]
         ): Vnode<Attrs, State>;
+        /** Creates a virtual element (Vnode). */
+        (selector: string, attributes: Attributes, ...children: Children[]): Vnode<any, any>;
+        /** Creates a virtual element (Vnode). */
+        <Attrs, State>(component: ComponentTypes<Attrs, State>, ...args: Children[]): Vnode<Attrs, State>;
```
Comment (dead-claudia) [inline types/mithril/index.d.ts:54]: Should this also be moved up, or does it still autocomplete correctly in the position it is here?

**Comment by dead-claudia** [comment]
Note to DT maintainers: this is intended to alter autocomplete only, and this was confirmed off repo. The lack of test changes is expected.

File: types/mithril/index.d.ts (hunk 1 above)
Comment (foxdonut) [reply types/mithril/index.d.ts:54]: It should not be moved up, the important thing here is that the functions that take more parameters come before those with fewer parameters.

**Review by dead-claudia — APPROVED** [review]

**Review by gabritto — APPROVED** [review]

**Pull request merged**